src/xpk/commands/cluster.py

```diff
@@ -177,6 +178,11 @@ def cluster_create(args) -> None:
   if create_cluster_configmaps_code != 0:
     xpk_exit(create_cluster_configmaps_code)
 
+  if args.enable_ray_cluster:
+    return_code = install_ray_cluster(args, system)
+  if return_code != 0:
```

Comment:
what if args.enable_ray_cluster is not defined? Shall we check return code than?

Reply:
Oh, whoops. Meant to indent 183-184